wrappers/swift/SDWLogConfiguration.swift

```diff
@@ -0,0 +1,214 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Class wrapping `ODWLogConfiguration` ObjC class object, representing configuration related to events.
+public class LogConfiguration {
+    private var odwLogConfiguration: ODWLogConfiguration
+
+    /**
+    Constructs `self` with provided config as `ODWLogConfiguration` object.
+
+    - Parameters:
+        - config: `ODWLogConfiguration` object which would be wrapped to.
+    */
+    public init(config: ODWLogConfiguration) {
```

Comment:
Do this need to be public? I assume this is only used by getLogConfigurationCopy() ?

Reply:
Just tried to follow the similar strategy as what ObjC was following with cpp, as objc is exposing a constructor with cpp LogConfiguration so i kept it public.